src/Equations/AnalyticDistribution.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * Functions to evaluate the analytic pitch-angle distribution. 
+ * (maybe other analytic distributions will be added later?)
+ */
+
+#include "DREAM/Equations/AnalyticDistribution.hpp"
+#include "FVM/config.h"
+#include "DREAM/Equations/EffectiveCriticalField.hpp"
+#include "DREAM/Equations/RunawayFluid.hpp"
+
+using namespace DREAM;
+
+/**
+ * Constructor.
+ *
+ */
+AnalyticDistribution::AnalyticDistribution(FVM::RadialGrid *rGrid, PitchScatterFrequency *nuD, OptionConstants::collqty_Eceff_mode Eceff_mode) 
+: rGrid(rGrid), nuD(nuD), Eceff_mode(Eceff_mode){}
+
+// ok, maybe re-name the constant so it doen't belong to Eceff?
+real_t AnalyticDistribution::evaluatePitchDistribution(len_t ir, real_t xi0, real_t p, 
+real_t Eterm, CollisionQuantity::collqty_settings *inSettings, gsl_integration_workspace *gsl_ad_w){
```

Comment:
I find it a bit confusing to split the argument list in this way (with same indentation for arguments and method head). I would rather write it as
```
real_t AnalyticDistribution::evaluatePitchDistribution(
    len_t ir, real_t xi0, real_t p, real_t Eterm,
    CollisionQuantity::collqty_settings *inSettings, gsl_integration_workspace *gsl_ad_w
) {
```

Reply:
@Embreus wrote this function, so I would suggest he fixes it